includes/Core/Admin/Screen.php

```diff
@@ -230,9 +230,13 @@ public function enqueue_assets( Assets $assets ) {
 		// Enqueue base admin screen stylesheet.
 		$assets->enqueue_asset( 'googlesitekit-admin-css' );
 
-		if ( $this->args['enqueue_callback'] ) {
-			call_user_func( $this->args['enqueue_callback'], $assets );
-		}
+		$cb = isset( $this->args['enqueue_callback'] ) && is_callable( $this->args['enqueue_callback'] )
```

Comment:
I think we can simplify this to remove the `isset` check since the key will always exist and `is_callable` should be fine to use with `null`, no?

Reply:
Makes sense, updated. `null` is not callable.